browser/src/control/Control.UIManager.js

```diff
@@ -581,9 +581,13 @@ L.Control.UIManager = L.Control.extend({
 			if (item.id === button.id)
 				return;
 		}
-
 		this.customButtons.push(button);
-		this.insertCustomButton(button);
+		if (button.tablet === undefined) {
+			button.tablet = true; // setting default value as true so in future we can use tablet prop of insert button
+		}
+		if (button.tablet || !window.mode.isTablet()) {
```

Comment:
Is this a typo? if button.tablet is true or if isTablet is not true

Reply:
no it's not a typo. 
basicallly is button.tablet is set to true then we will insertCustom button 
And if button.tablet is set to false then we will check if current resolution is desktop then in this case we also need to saw the button 

only if button.tab = false and resolution is also for tablet then and then we should remove the insert button